src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -269,10 +269,14 @@ class FluxPipeline : public DiffusionPipeline {
 
         ov::Tensor latent_image_ids = prepare_latent_image_ids(generation_config.num_images_per_prompt, height / 2, width / 2);
 
+        ov::Tensor guidance = ov::Tensor(ov::element::f32, {generation_config.num_images_per_prompt});
+        std::fill_n(guidance.data<float>(), guidance.get_size(), static_cast<float>(generation_config.guidance_scale));
+
         m_transformer->set_hidden_states("pooled_projections", pooled_prompt_embeds);
         m_transformer->set_hidden_states("encoder_hidden_states", prompt_embeds);
         m_transformer->set_hidden_states("txt_ids", text_ids);
         m_transformer->set_hidden_states("img_ids", latent_image_ids);
+        m_transformer->set_hidden_states("guidance", guidance);
```

Comment:
should we add it based on `guidance_embeds` flag ?

Reply:
fixed